ckan/public/robots.txt

```diff
@@ -1,6 +0,0 @@
-User-agent: *
```

Comment:
There could be sites that have started to rely on this version of the file. Is there a way to accommodate both?

Reply:
@ThrawnCA what are you thinking? A config option to disable the robots view?

Reply:
I was more thinking "use whichever robots file is present", possibly with the `public` version being prioritised if present (since extensions can decide for themselves whether or not to supply it).

Just dropping it is fine, so long as it's noted as a breaking change.

Reply:
@wardi should i remove it? 

Thanks